cmd/rad/cmd/root.go

```diff
@@ -153,6 +154,16 @@ func prettyPrintJSON(o any) (string, error) {
 	return string(b), nil
 }
 
+// handlePanic is the global panic recovery handler that formats and displays panic information
+func handlePanic() {
+	if r := recover(); r != nil {
+		fmt.Printf("Error: An unexpected internal error occurred: %v\n\n", r)
+		fmt.Println(string(debug.Stack()))
+		fmt.Println("\nPlease report this issue at https://github.com/radius-project/radius/issues")
+		fmt.Println("") // Output an extra blank line for readability
```

Comment:
nit: This part mixed code for the same purpose - new line. We have `\n` and `fmt.Println("")`. I think consistancy is the key, especially from code readability perspective.

```suggestion
		photoprints("Error: An unexpected internal error occurred: %v\n", r)
		fmt.Println("") // Output an extra blank line for readability
		fmt.Println(string(debug.Stack()))
		fmt.Println("") // Output an extra blank line for readability
		fmt.Println("Please report this issue at https://github.com/radius-project/radius/issues")
		fmt.Println("") // Output an extra blank line for readability
```